client/src/layout/SettingCompany.js

```diff
@@ -0,0 +1,169 @@
+  },
+  buttonAdd: {
+    borderRadius: theme.shape.borderRadius * 4,
+    width: "4em",
+    height: "2em",
+    marginRight: "1em",
+  },
+  buttonSave: {
+    position: "relative",
+    borderRadius: theme.shape.borderRadius * 4,
+    marginTop: theme.spacing(5),
+    width: "10em",
+    height: "3em",
+  },
+}));
+
+const SettingCompany = () => {
+  const classes = useStyles();
+
+  const [company, setCompany] = useState("Company ABC");
+  const [value, setValue] = useState("");
+
+  const updateHandler = () => {
+    setCompany(value);
+    setValue("");
+  };
+
+  // handle subscriber info update submit
+  const submitHandler = (e) => {
+    e.preventDefault();
+  };
+
+  return (
+    <>
+      <Grid fullWidth container spacing={2}>
+        <Grid container xs={2} direction="column" justify="space-between">
+          <Grid item>
+            <Typography variant="h4" style={{ paddingTop: "15px" }}>
+              Your company
+            </Typography>
+          </Grid>
+          <Grid item>
+            <Typography variant="h4" style={{ paddingBottom: "30px" }}>
+              Weekly report
+            </Typography>
+          </Grid>
+        </Grid>
+        <Grid item xs={6}>
+          <form id="company-form" onSubmit={submitHandler}>
+            <FormControl variant="filled" className={classes.inputBox}>
+              <InputLabel htmlFor="filled-company-name"></InputLabel>
+              <InputBase
+                id="filled-company-name"
+                placeholder="Company name"
+                value={company}
+                fullWidth
+                classes={{ input: classes.input, root: classes.inputBase }}
+                endAdornment={
+                  <InputAdornment position="end">
+                    <Button
+                      variant="text"
+                      onClick={() => setCompany("")}
+                      className={classes.buttonRemove}
+                    >
+                      REMOVE
+                    </Button>
+                  </InputAdornment>
+                }
+                inputProps={{ "aria-label": "Company name" }}
+              />
+            </FormControl>
+            <FormControl className={classes.inputBox} focused>
+              <InputLabel htmlFor="add-company-name"></InputLabel>
+              <InputBase
+                id="add-company-name"
+                placeholder="Company name"
+                fullWidth
+                classes={{ input: classes.input, root: classes.inputBase }}
+                value={value}
+                onChange={(e) => setValue(e.target.value)}
+                endAdornment={
+                  <InputAdornment position="end">
+                    <Button
+                      variant="contained"
+                      color="primary"
+                      onClick={updateHandler}
+                      className={classes.buttonAdd}
+                    >
+                      ADD
+                    </Button>
```

Comment:
Adding a company deletes the original, should I have the option to have multiple company names?

Reply:
Yeah, we are going to have that function later. Jess said just comment out the add function for now